src/server/pps/server/api_server.go

```diff
@@ -2342,6 +2329,25 @@ func (a *apiServer) GarbageCollect(ctx context.Context, request *pps.GarbageColl
 	if err := checkLoggedIn(pachClient); err != nil {
 		return nil, err
 	}
+	// Get all objects referenced by pipeline tags
```

Comment:
I think this comment was copy/pasted ... I don't think it really makes sense here.

Reply:
Indeed it was.